packages/node_modules/pouchdb-utils/src/cloneBinaryObject-browser.js

```diff
@@ -14,14 +14,8 @@ function cloneBinaryObject(object) {
   if (object instanceof ArrayBuffer) {
     return cloneArrayBuffer(object);
   }
```

Comment:
```suggestion
  if (object instanceof ArrayBuffer) {
    return object.slice();
  }
```
Since we dropped support for IE: `cloneArrayBuffer` can be removed too 😉

Side note: `start` (first parameter) of .slice is optional and has 0 as default, calling without any parameter is equivalent to .slice(0, size).

Reply:
is blob.slice() guaranteed to work?  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer#transferring_arraybuffers

Reply:
It's in the specs:
https://tc39.es/ecma262/multipage/structured-data.html#sec-arraybuffer.prototype.slice

And the compatibility table on Mozilla says, that every browser supports it:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice#browser_compatibility

Same  results on caniuse (even was available in IE 11... So this workaround is ancient):
https://caniuse.com/?search=Arraybuffer%20slice

Reply:
I was referring to this:

> Detached buffers have the following behaviors:
> ...
> Methods, such as ... [slice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/slice), throw a [TypeError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError) when invoked

I don't currently have enough context to confidently make changes to `ArrayBuffer` usage.  I can see how it could be a good follow-up to this PR though.

Reply:
> I can see how it could be a good follow-up to this PR though.

Sounds like a plan.